app/models/user.rb

```diff
@@ -138,4 +138,8 @@ def full_title
       title.presence || affiliation.presence
     end
   end
+
+  def to_xapi_agent
```

Comment:
That's kind of serialization - from `User` model to xAPI agent - it should not be in user model. I would rather have it as a separate class that would build agent from email & name. We should know nothing more about user here. We do not need that.